src/utility/firebase.js

```diff
@@ -92,3 +97,51 @@ export const getProject = async (user_id, setProjectCallback, setFeedbackCallbac
       })
   }
 }
+
+const createNewApplication = async user => {
+  const userId = {
+    _id: user.uid,
+  }
+  const basicInfo = {
+    basicInfo: {
+      email: user.email,
+      firstName: user.displayName.split(' ')[0],
+      lastName: user.displayName.split(' ')[1],
+    },
+  }
+  const submission = {
+    submission: {
+      lastUpdated: firebase.firestore.Timestamp.now(),
+      submitted: false,
+    },
+  }
+
+  const newApplication = {
+    ...hackerApplicationTemplate,
+    ...basicInfo,
+    ...submission,
+    ...userId,
+  }
+
+  await applicantsRef.doc(user.uid).set(newApplication)
+}
+
```

Comment:
Also was wondering if you can make a simple getter for checking if applicant has responded or not? Just because the dashboard has a view for un-RSVP and no RSVP
```
export const getUserResponse = async user => {
  const applicant = await applicantsRef.doc(user.uid).get()
  if (!applicant.exists) {
      return false;
  }
  return applicant.data().status.responded
}
```

Reply:
mm I think this should be part of the status, not a separate function, thoughts?

Reply:
Oh actually sure I think that works too!
I think @andykwan1124 might need these for the dashboard, e.g.
- un-RSVPed: `responded && !attending`
- no RSVP or not attending: `!responded && !attending`

Reply:
will include in my status update PR!